app/pages/BeelieversAuction/BeelieversBid.tsx

```diff
@@ -39,15 +51,34 @@ export function BeelieversBid({ leaderBoardData = [] }: BeelieversBidProps) {
 	const bidForm = useForm<BeelieversBidForm>({
 		mode: "all",
 		reValidateMode: "onChange",
+		disabled: isPending || isSuccess || isError,
 		defaultValues: {
-			bid: "",
+			bid: "", // from SUI to MIST
 			note: "",
 		},
 	});
-	const { handleSubmit } = bidForm;
+	const { handleSubmit, watch, trigger, reset, setValue } = bidForm;
 
 	if (suiAddr == null) return <SuiModal />;
 
+	const suiBid = watch("bid");
+
+	const mistBidAmount: bigint = parseSUI(suiBid?.length > 0 && suiBid !== "." ? suiBid : "0");
+
+	// Why do we need this
+	// const resetForm = useCallback(() => {
+	// 	resetMutation();
+	// 	reset({
+	// 		bid: "",
+	// 	});
+	// }, [reset, resetMutation]);
+
+	// useEffect(() => {
+	// 	if (suiBid) {
+	// 		trigger();
+	// 	}
+	// }, [isSuiWalletConnected, suiBid, trigger]);
```

Comment:
@rmeena840 could you respond on this?

Reply:
this is needed to reset the form to default values after placing bid.